flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adapter/DefaultResultPartitionTest.java

```diff
@@ -34,24 +33,18 @@
 /** Unit tests for {@link DefaultResultPartition}. */
 public class DefaultResultPartitionTest extends TestLogger {
 
-    private static final TestResultPartitionStateSupplier resultPartitionState =
-            new TestResultPartitionStateSupplier();
-
-    private final IntermediateResultPartitionID resultPartitionId =
-            new IntermediateResultPartitionID();
-    private final IntermediateDataSetID intermediateResultId = new IntermediateDataSetID();
+    @Test
```

Comment:
There is no need to make these changes

Reply:
Thank you for pointing this out. Removed.